app/src/main/java/edu/nextstep/camp/common/UiText.kt

```diff
@@ -0,0 +1,16 @@
+package edu.nextstep.camp.common
+
+import android.content.Context
+import androidx.annotation.StringRes
+
+sealed class UiText {
+	data class DynamicText(val text: String): UiText()
+	data class StringResource(@StringRes val resId: Int, val formatArgs: List<Any> = emptyList()): UiText()
```

Comment:
```suggestion
	data class StringResource(@StringRes val resId: Int): UiText()
```
미사용 파라미터는 제거해보면 어떨까요?

Reply:
@BeokBeok 해당 파라미터는 추후에 문자열 리소스를 인자들과 함께 사용해서 가져와야 할때를 대비해서 추가해두었습니다.

Reply:
@ChanJun-Park 
미리 대비해두는 것은 좋습니다.
하지만 추후에 사용할 것 같아서 추가했다가 사용하지 않게되는 경우가 의외로 많고, 사용하지 않게되면 레거시로 남게 됩니다.
가급적이면 사용을 할 때 추가하는 습관을 들여보면 어떨까요?